src/prompt_tasks.py

```diff
@@ -135,4 +145,5 @@ def not_allowed():
 The executor must not use a pre-trained model unless it is explicitly mentioned in the description.
 The executor must not train a model.
 The executor must not use any attribute of Document accept Document.text.
+Note that the Dockerfile only has access to the files: executor.py, requirements.txt, config.yml, test_executor.py.
```

Comment:
separate them into must not for executor and for Dockerfile

Reply:
done